database/migration/V20161226_0927__createAttendees.sql

```diff
@@ -41,13 +41,13 @@ CREATE TABLE `attendee_requested_collaborators` (
 		ON UPDATE NO ACTION
 );
 
-CREATE TABLE `attendee_extra_infos` (
+CREATE TABLE `attendee_longform` (
```

Comment:
See above about model usage.

Reply:
Fixed in cd52938e2c67991a451212c0cd3d2fa14e1edbe7